lib/reltool/src/reltool_target.erl

```diff
@@ -201,11 +197,18 @@ do_gen_config(#mod{name = Name,
 	_ ->
 	    []
     end;
-do_gen_config(#rel{name = _Name,
-		   vsn = _Vsn,
-		   rel_apps = RelApps},
-	      InclDefs) ->
-    [do_gen_config(RA, InclDefs) || RA <- RelApps];
+do_gen_config(#rel{name = Name,
+                   vsn = Vsn,
+                   rel_apps = RelApps,
+                   load_dot_erlang = LoadDotErlang},
+              InclDefs) ->
+    RelAppsConfig = [do_gen_config(RA, InclDefs) || RA <- RelApps],
+    if
+        LoadDotErlang =:= true ->
```

Comment:
To be correct, I think we need to do this only if `InclDefs=:=false`. That is, we should return the options list, not matter the value of load_dot_erlang, if `InclDefs=:=true`. This will of course open for more compatibility issues, but I think we should do it anyway. I'll mark the change as "potentially incompatible" in the release notes.

Reply:
I added the most recent commit to ensure the options list is present with the default when `InclDefs=:=true`.  Does everything looks good?

Reply:
Yes, it looks good. I'll include it in tests today.